### PR TITLE
[public-api] Proxy Cookies to downstream for auth

### PR DESCRIPTION
**Pull request by easyCZ**
## Description

Cookies on the request are sent downstream to Server to use as authentication. We support both Access Tokens, and Cookies.

## Related Issue(s)

Fixes #

## How to test

Unit tests

See it on dashboard in [this PRs preview](https://github.com/gitpod-io/gitpod/pull/14403).

## Release Notes

```release-note
NONE
```

## Werft options:

- [ ] /werft with-local-preview
      If enabled this will build `install/preview`
- [x] /werft with-preview
- [ ] /werft with-large-vm
- [ ] /werft with-integration-tests=all
      Valid options are `all`, `workspace`, `webapp`, `ide`

### CODE REVIEW

File: install/installer/pkg/components/public-api-server/configmap.go [hunk 1]
```diff
@@ -35,7 +35,7 @@ func configmap(ctx *common.RenderContext) ([]runtime.Object, error) {
 	})
 
 	cfg := config.Configuration{
-		GitpodServiceURL:               fmt.Sprintf("wss://%s/api/v1", ctx.Config.Domain),
+		GitpodServiceURL:               fmt.Sprintf("wss://%s/api/gitpod", ctx.Config.Domain),
```
Comment (geropl) [inline install/installer/pkg/components/public-api-server/configmap.go:38]: Why mthis change? More of a drive-by, or was it required for some reason? :thinking:

**Review by geropl — APPROVED** [review]
Code LGTM, unit tests and manual test worked like a charm :a: :a: :a: :rocket:

File: install/installer/pkg/components/public-api-server/configmap.go (hunk 1 above)
Comment (easyCZ) [reply install/installer/pkg/components/public-api-server/configmap.go:38]: The other endpoint doesn't accept cookies

Comment (geropl) [reply install/installer/pkg/components/public-api-server/configmap.go:38]: Ah, thansk for refreshing my memory.
